transaction.go

```diff
@@ -369,6 +379,131 @@ func (t *Transaction) Encode() []byte {
 	return mustRLPEncode(&temp)
 }
 
+// DecodeTransactionPayloadMessage returns the signable message for the transaction payload.
```

Comment:
The doc comments for the decode functions are wrong I think 

Maybe something like this?

> DecodeTransactionPayloadMessage constructs a transaction from an RLP-encoded payload message.